packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/node-pool.class.js

```diff
@@ -0,0 +1,21 @@
+export default class NodePool {
+  constructor(pool) {
+    Object.assign(this, pool);
+  }
```

Comment:
NP: please use explicit object properties (to avoid errors in future).

```suggestion
  constructor({ status }) {
    Object.assign(this, { status });
  }
```